modules/openxr/doc_classes/OpenXRAPIExtension.xml

```diff
@@ -30,6 +30,13 @@
 				Returns an error string for the given [url=https://registry.khronos.org/OpenXR/specs/1.0/man/html/XrResult.html]XrResult[/url].
 			</description>
 		</method>
+		<method name="get_hand_tracker_handle">
+			<return type="int" />
+			<param index="0" name="hand_index" type="int" />
+			<description>
+				Returns the corresponding XRHandTrackerEXT handle for the given hand index value.
```

Comment:
It's an internal class so this may be one way to make that clearer.
```suggestion
				Returns the corresponding [code]XRHandTrackerEXT[/code] handle for the given hand index value.
```

Reply:
@Mickeon while I believe it's a pointer beneath the surface, I'd argue it still makes more sense to refer to it as a handle. It's only ever referred to as such in the [OpenXR spec](https://registry.khronos.org/OpenXR/specs/1.0/html/xrspec.html) (feel free to search for any mentions of it), so why start calling it a pointer here?

Reply:
That's alright then. It matches the standard specification.